src/types.ts

```diff
@@ -45,6 +45,7 @@ export interface Schema<Data> {
 
 export type Options<FormValues extends FieldValues = FieldValues> = Partial<{
   mode: Mode;
+  reValidateMode: Mode;
```

Comment:
This typing is incorrect. This results in `'onSubmit' | 'onChange' | 'onBlur'`.

In order for this to be correct and use the `keyof EVENTS`, you will have to change the constant to 

```
const EVENTS = {
  blur: 'blur',
  change: 'change',
  input: 'input'
}
```

Then this will have to be 

```suggestion
  reValidateMode: keyof EVENTS;
```